omniport/core/base_auth/managers/get_user.py

```diff
@@ -55,13 +55,15 @@ def get_user(username):
         q_email_address = Q(email_address=username)
         q_institute_webmail_address = Q(institute_webmail_address=username)
         q = (
-                q_primary_phone_number
-                | q_email_address
-                | q_institute_webmail_address
+            q_primary_phone_number
+            | q_secondary_phone_number
+            | q_email_address
+            | q_institute_webmail_address
```

Comment:
Sure about this? I think code in brackets needs to be one extra indent level.

Reply:
yes autopep8 did auto formatting on this so i guess they are correct